lib/command_line_reporter.rb

```diff
@@ -63,7 +63,7 @@ def footer(options = {})
   end
 
   def horizontal_rule(options = {})
-    validate_options(options, :char, :width, :color, :bold)
+    validate_options(options, :char, :width, :color, :color_code, :bold)
```

Comment:
The more I look at this code I have a problem with 2 options instead of just overloading the original :color option with the specification of "red" or "ff0000"